gradle-plugin/src/main/java/com/contrastsecurity/gradle/plugin/ContrastGradlePlugin.java

```diff
@@ -1,21 +1,199 @@
 package com.contrastsecurity.gradle.plugin;
 
+import com.contrastsecurity.gradle.plugin.extensions.ContrastConfigurationExtension;
+import com.contrastsecurity.sdk.ContrastSDK;
+import com.contrastsecurity.sdk.UserAgentProduct;
+import java.io.File;
+import java.time.Instant;
+import java.time.ZoneId;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Locale;
 import org.gradle.api.Plugin;
 import org.gradle.api.Project;
+import org.gradle.api.Task;
+import org.gradle.api.tasks.TaskProvider;
+import org.gradle.api.tasks.testing.Test;
+import org.jetbrains.annotations.VisibleForTesting;
 
 /**
  * Gradle plugin for contrast utilities. The goals for this plugin are defined here <a
  * href=https://contrast.atlassian.net/browse/JAVA-8252>JAVA-8252</a>
  */
 public class ContrastGradlePlugin implements Plugin<Project> {
 
+  public static ContrastSDK SDK_INSTANCE;
```

Comment:
This should not be static and I don't even think you need a field.  There's nothing stopping a customer from using the plugin in multiple projects where one project might have different extension values.  Each should be completely unique and pinned to the plugin instance of the specific project.